src/shippingservice/main.go

```diff
@@ -90,6 +90,10 @@ func (s *server) Check(ctx context.Context, req *healthpb.HealthCheckRequest) (*
 	return &healthpb.HealthCheckResponse{Status: healthpb.HealthCheckResponse_SERVING}, nil
 }
 
+func (s *server) Watch(req *healthpb.HealthCheckRequest, ws healthpb.Health_WatchServer) error {
+	return nil
```

Comment:
Can we return a proper grpc error here instead so that if somehow this is invoked in the future, we don't ignore it. http://avi.im/grpc-errors/#go

    return status.Errorf(codes.Unimplemented, "health check via Watch not implemented")